src/components/pages/SearchResults/SearchResultsPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
For clarity perhaps we should rename this component as SearchResultsCard

Reply:
I would normally call it "card" but the search bar is rendered there, as well, which is static on the page. If you still want me to change it, though, I will.

Reply:
There we go, I see what you were talking about. I fixed the nomenclature of the files and file structure.